v22.1/ui-queues-dashboard.md

```diff
@@ -0,0 +1,189 @@
+
+The **Replica GC Queue** graph shows you various details about the health and performance of the replica garbage collection queue.
+
+On hovering over the graph, the values for the following metrics are displayed:
+
+Metric | Description
+--------|----
+Successful Actions | The number of replicas successfully processed by the replica garbage collection queue, as tracked by the `queue.replicagc.process.success` metric.
+Pending Actions | The number of pending replicas in the replica garbage collection queue, as tracked by the `queue.replicagc.pending` metric.
+Replicas Removed | The number of replica removals attempted in the replica garbage collection queue, as tracked by the `queue.replicagc.removereplica` metric.
+
+## Replication Queue
+
+<img src="{{ 'images/v22.1/ui_replication_queue.png' | relative_url }}" alt="DB Console replication queue graph" style="border:1px solid #eee;max-width:100%" />
+
+The **Replication Queue** graph shows you various details about the health and performance of the replication queue.
+
+On hovering over the graph, the values for the following metrics are displayed:
+
+Metric | Description
+--------|----
+Successful Actions | The number of replicas successfully processed by the replication queue, as tracked by the `queue.replicate.process.success` metric.
+Pending Actions | The number of pending replicas in the replication queue, as tracked by the `queue.replicate.pending` metric.
+Replicas Added | The number of replica additions attempted by the replication queue, as tracked by the `queue.replicate.addreplica` metric.
+Replicas Removed | The number of replica removals attempted by the replication queue, as tracked by the `queue.replicate.removereplica` metric.
+Dead Replicas Removed | The number of dead replica removals attempted by the replication queue, as tracked by the `queue.replicate.removedeadreplica` metric.
+Learner Replicas Removed | The number of learner replica removals attempted by the replication queue, as tracked by the `queue.replicate.removelearnerreplica` metric.
+Replicas Rebalanced | The number of replica rebalancer-initiated additions attempted by the replication queue, as tracked by the `queue.replicate.rebalancereplica` metric.
+Leases Transferred | The number of range lease transfers attempted by the replication queue, as tracked by the `queue.replicate.transferlease` metric.
+Replicas in Purgatory | The number of replicas in the replication queue's purgatory currently awaiting allocation options, as tracked by the `queue.replicate.purgatory` metric.
+
+## Split Queue
+
+<img src="{{ 'images/v22.1/ui_split_queue.png' | relative_url }}" alt="DB Console split queue graph" style="border:1px solid #eee;max-width:100%" />
+
+The **Split Queue** graph shows you various details about the health and performance of the split queue.
+
+On hovering over the graph, the values for the following metrics are displayed:
+
```

Comment:
Same suggestion as the previous table, might be more readable with the full metric name split into its own column so they are aligned vertically.